src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -80,4 +80,25 @@ export class CourseManagementRequests {
     addStudentToCourse(courseId: number, studentName: string) {
         return cy.request({ url: COURSE_BASE + courseId + '/students/' + studentName, method: POST });
     }
+
+    createModelingExercise(modelingExercise: string) {
+        return cy.request({
+            url: '/api/modeling-exercises',
+            method: 'POST',
+            body: modelingExercise,
+            headers: {
+                Authorization: 'Bearer ' + Cypress.env(authTokenKey),
+            },
```

Comment:
The authorization header can be removed since it will be added automatically by our request command